src/app.tsx

```diff
@@ -53,8 +54,26 @@ interface Alert {
     actionLinks?: React.ReactNode
 }
 
+const get_path = (options: Location['options']) => {
+    let currentPath = decodeURIComponent(options.path?.toString() || "/");
+
+    // Trim all trailing slashes
+    currentPath = currentPath.replace(/\/+$/, '');
+
+    // Our path will always be `/foo/` formatted
+    if (!currentPath.endsWith("/")) {
```

Comment:
Won't this always be true because of the regex above? But I suppose this makes it more readable :)

Reply:
Good point, and thinking about it. This should have some qunit tests! Like for example starter-kit has

Reply:
Thinking about it, it does make sense as regex is hard ;-)  Maybe we should instead just shortcut the `/` condition and directly return.
```
> '/'.replace(/\/+$/, '')
''
```

Reply:
or you could do `.replace(/\/+$/, '/')` to replace multiple slashes at the end with a single one but I am fine with the way how it is now